ScrubBot.Modules/AdminModule.cs

```diff
@@ -13,9 +13,11 @@ namespace ScrubBot.Modules
     [RequireUserPermission(GuildPermission.Administrator, Group = nameof(AdminModule)), RequireOwner(Group = nameof(AdminModule))]
     public class AdminModule : Module
     {
-        public AdminModule()
-        {
+        DiscordSocketClient _client;
 
+        public AdminModule(DiscordSocketClient client)
```

Comment:
Client can be accessed with Context.Client
```suggestion
        public AdminModule()
```